docs/source/releases/latest/documentation-AMSR2_NetCDF_reader.yaml

```diff
@@ -0,0 +1,20 @@
+refactor:
```

Comment:
Do not merge this until https://github.com/NRLMMD-GEOIPS/geoips/pull/850 and https://github.com/NRLMMD-GEOIPS/geoips/pull/840 are merged - they fix the CI that is currently failing on this PR

Reply:
@biosafetylvl5 The attached log in the PR contains the relevant tests from the integration tests, all of which have passed successfully.

Reply:
Which test did you chose? It doesn't look like all of the integration tests were run

Reply:
all of the amsr2 tests.


"$GEOIPS_PACKAGES_DIR/geoips/tests/scripts/amsr2.tc.89H-Physical.imagery_annotated.sh"
"$GEOIPS_PACKAGES_DIR/geoips/tests/scripts/amsr2_ocean.tc.windspeed.imagery_clean.sh"
"$GEOIPS_PACKAGES_DIR/geoips/tests/scripts/amsr2.config_based_overlay_output.sh"
"$GEOIPS_PACKAGES_DIR/geoips/tests/scripts/amsr2.config_based_overlay_output_low_memory.sh" \